.env.example

```diff
@@ -3,10 +3,17 @@ NEXT_PUBLIC_BASE_URL=https://api.turnkey.com
 NEXT_PUBLIC_ORGANIZATION_ID=<your-organization-id>
 NEXT_PUBLIC_RP_ID=<your-rp-id>
 NEXT_PUBLIC_GOOGLE_OAUTH_CLIENT_ID=<your-google-oauth-client-id>
+NEXT_PUBLIC_APPLE_OAUTH_CLIENT_ID=<your-apple-oauth-client-id>
+NEXT_PUBLIC_FACEBOOK_CLIENT_ID=<your-facebook-oauth-client-id>
+NEXT_PUBLIC_FACEBOOK_GRAPH_API_VERSION=21.0
+NEXT_PUBLIC_FACEBOOK_AUTH_VERSION=11.0
 # Sensitive values ideally for server only
 
 ALCHEMY_API_KEY=
 COINGECKO_API_KEY=
 
 TURNKEY_API_PUBLIC_KEY=
-TURNKEY_API_PRIVATE_KEY=
\ No newline at end of file
+TURNKEY_API_PRIVATE_KEY=
+
+# randomly generated secret
```

Comment:
should we mention how to use `src/lib/facebook-utils.ts` somewhere?

Reply:
Like in the README? It's an internal implementation detail of the callback FYI. But happy to provide comments or docs if you think that would be helpful!

Reply:
this is coming from someone with little to no context on how facebook oauth is implemented, but as a user of this example, how can I generate this value (FACEBOOK_SECRET_SALT)?

Reply:
Ah, yes! Makes sense.